doc/classes/InputEventJoypadButton.xml

```diff
@@ -16,9 +16,8 @@
 		<member name="pressed" type="bool" setter="set_pressed" getter="is_pressed" default="false">
 			If [code]true[/code], the button's state is pressed. If [code]false[/code], the button's state is released.
 		</member>
-		<member name="pressure" type="float" setter="set_pressure" getter="get_pressure" default="0.0" is_deprecated="true">
+		<member name="pressure" type="float" setter="set_pressure" getter="get_pressure" default="0.0" deprecated="This property is never set by the engine and is always [code]0[/code].">
 			Represents the pressure the user puts on a pressure-sensitive button.
```

Comment:
I think this was discussed recently in a PR by @Mickeon, but I still think we shouldn't be documenting what a property _ought to do_ if it doesn't do it. If this is never set, that's everything the user needs to know, not what some potential thirdparties may decide to represent with this unused property.

Reply:
Because the **Deprecated** message comes first in the PR now, this whole description to me is off. I don't think saying "Represents the pressure[...]" makes sense anymore because it feels a bit... contradictory? At best it should be "Should represent the pressure[...]". At worst, it should not exist at all.